helm/alfresco-content-services/values.yaml

```diff
@@ -146,6 +146,39 @@ libreoffice:
     maxTransforms: 99999
     maxTransformSeconds: 1800
 
+# Declares the alfresco-libreoffice service used by the content repository
```

Comment:
Trivial: Declares the alfresco-tika service ...

Reply:
fixed